Procfile

```diff
@@ -0,0 +1 @@
+web: bundle exec rake cf:on_first_instance db:migrate && bundle exec puma
```

Comment:
To tell Puma to use the settings defined in `config/puma.rb`, you need to add `-C ./config/puma.rb` at the end of this line.